serverless-heaven/serverless-webpack
# Fix eslint prettier

## What did you implement:

Closes #515



## How did you implement it:
- Configure [prettier-eslint](https://github.com/prettier/prettier-eslint-cli) to format code, remove eslint-config-prettier 
- Update prettier config to be closer to existing code
- Move `npm test` command out of `lint-staged` to `.huskyrc`. This was causing errors trying to run staged files as if they were tests. All tests should be run not just the staged test files.
- Reformat code with [prettier-eslint](https://github.com/prettier/prettier-eslint-cli)


## How can we verify it:
- @HyperBrain Look at the last commit in this PR and make sure that the code style is acceptable since introducing prettier normalizes spacing, semicolons, line-length, etc. I tried to make the config as close to the current style preferences but the current ESLint rules weren't tight enough to enforce consistency code being reformatted was inevitable.
- run `npm run format` then `npm run eslint` and verify there are no formatting lint issues
- Try committing some code then run `npm run eslint` and ensure there are no issues.



## Todos:

- [x] Make sure code coverage hasn't dropped (Some existing lines were split that contained uncovered code this increased the number of uncovered lines but didn't affect branches or functions).

***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO
